packages/@cdklabs/typewriter/src/renderer/typescript.ts

```diff
@@ -411,11 +411,11 @@ export class TypeScriptRenderer extends Renderer {
     throw new Error(`Symbol ${sym} (in ${sym.scope}) not visible from ${this.scopes[0]} (missing import?)`);
   }
 
-  protected renderFunction(func: CallableDeclaration) {
+  protected renderFunction(func: FreeFunction) {
```

Comment:
This interface change makes be a little anxious. Are you sure this won't cause any issues?
`renderFunction` is protected, so we should be able to reason about this question from the code paths.